static/js/components/CourseList.js

```diff
@@ -44,6 +54,13 @@ class CourseList extends React.Component {
           </li>
         </ul>
         {table}
+      </div>
+      <div className="clear-float">
+        <br/>
+        <p>You need to pass {totalCourses - coursesPassed} more courses
+          before you can apply for <strong>{program.title}</strong> Master's Degree.</p>
+        <Button bsStyle="primary" disabled>Apply for <strong>{program.title}</strong> Master Degree</Button>
+      </div>
       </div>;
```

Comment:
additionally I prefer this style for JSX returns:

```js
return (
  <div id="foo">
    { content }
  </div>
);
```

but we seem to also use this style:

```js
return <div>
  { content }
</div>
```

so if you prefer that I think it's a matter of preference